http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/auth/LeoAuthProviderHelperSpec.scala

```diff
@@ -1,89 +0,0 @@
-package org.broadinstitute.dsde.workbench.leonardo.auth
-
-import akka.actor.ActorSystem
-import akka.http.scaladsl.model.StatusCodes.InternalServerError
-import akka.testkit.TestKit
-import org.broadinstitute.dsde.workbench.leonardo.CommonTestData
-import org.broadinstitute.dsde.workbench.leonardo.model.{NotebookClusterActions, ProjectActions}
-import org.broadinstitute.dsde.workbench.leonardo.service.ClusterNotFoundException
-import org.broadinstitute.dsde.workbench.model.UserInfo
-import org.broadinstitute.dsde.workbench.model.google.GoogleProject
-import org.scalatest.concurrent.ScalaFutures
-import org.scalatest.time.{Seconds, Span}
-import org.scalatest.{BeforeAndAfterAll, FlatSpecLike, Matchers}
-
-import scala.concurrent.{ExecutionContext, Future}
-import scala.concurrent.duration._
-
-/**
-  * Created by rtitle on 1/25/18.
-  */
-class LeoAuthProviderHelperSpec extends TestKit(ActorSystem("leonardotest")) with FlatSpecLike with Matchers with CommonTestData with ScalaFutures with BeforeAndAfterAll {
-  import system.dispatcher
-
-  override def afterAll(): Unit = {
-    TestKit.shutdownActorSystem(system)
-    super.afterAll()
-  }
-
```

Comment:
How are the exceptions (that used to be tested below) handled now? Do we no longer need map Sam exceptions to Leo exceptions?

Reply:
I think I prefer exception mapping to API errors done at leo route level (something like https://github.com/DataBiosphere/leonardo/pull/1073#discussion_r329683236) instead of throughout leo internals. Cuz if different layers in leo all try to translate errors, it's pretty confusing what the final error will be that we show to users